src/sentry/models/eventattachment.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BoundedBigIntegerField, FlexibleForeignKey, Model, sane_repr
+
+
+class EventAttachment(Model):
+    __core__ = False
+
+    project_id = BoundedBigIntegerField()
+    group_id = BoundedBigIntegerField(null=True)
+    event_id = models.CharField(max_length=32)
+    file = FlexibleForeignKey('sentry.File')
+    name = models.TextField()
+    date_added = models.DateTimeField(default=timezone.now)
```

Comment:
We can add this later, but probably useful to have an index on date_added if we did want to TTL?

Reply:
@mattrobenolt (project_id, date_added)?

Reply:
Yeah, if you'd want to have potentially different TTLs per project or something.

We can always add this later, so I don't think it's a big deal now to hypothesize what a use case might be.